src/components/Edit.js

```diff
@@ -44,40 +55,94 @@ export default class Edit extends Component {
 
   static propTypes = {
     onSubmit: PropTypes.func.isRequired,
+    tags: PropTypes.array.isRequired,
     note: PropTypes.object
   }
 
   static defaultProps = {
     note: {
       id: null,
-      inputBody: ''
+      inputBody: '',
+      tagIDs: [],
+      newTags: [],
     }
   }
 
   constructor(props) {
     super(props)
-    const { id, body } = props.note
-
+    const { id, body, tagIDs, newTags } = props.note
     this.state = {
       hasChanged: false,
       createMode: !props.note.id,
-      id: id,
-      inputBody: body
+      id: id || null,
```

Comment:
Musst du das explizit auf `null` setzen? Machst du das nicht ohnehin nur, wenn id nicht definiert ist?

Reply:
Ich übergebe später das Objekt an `App.saveNote()`. Dort frage ich spezifisch, ob `id === null`.
Das schien mir "kontrollierter" zu sein, als nur nach einem falsy Wert to fragen.

Im Branch "autosave" habe ich das jetzt sowieso geändert, da bekommen alle notes gleich von Anfang an eine ID.